rhizome/host/lib/vm_setup.rb

```diff
@@ -131,6 +131,13 @@ def hugepages(mem_gib)
   end
 
   def interfaces(nics, multiqueue)
+    # We first delete the network namespace for idempotency. Instead
+    # we could catch various exceptions for each command run, and if
+    # the error message matches certain text, we could resume. But
+    # the "ip link add" step generates the MAC addresses randomly,
+    # which makes it unsuitable for error message matching. Deleting
+    # and recreating the network namespace seems easier and safer.
+    r "ip netns del #{q_vm}"
```

Comment:
This is going to crash most of the time, which could be fine, except that case is not handled, right?